tests/lib/snaps/test-snapd-sh/bin/cmd

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+PS1='$ '
+command="$1"
+shift
+
+exec "$command" "$@"
```

Comment:
You could simplify the last three lines of this script to simply do `exec "$@"`.  No need to split off `$1`.

Reply:
Fair point. We carry a copy of this script in a couple of places. I can update them in a followup.